transform/runner_test.go

```diff
@@ -11,10 +12,24 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
-type fakePlugin func(u *unstructured.Unstructured) (PluginResponse, error)
+type fakePlugin struct {
+	Func func(u *unstructured.Unstructured) (PluginResponse, error)
+	name string
+}
+
+type fakePluginFunc func(u *unstructured.Unstructured) (PluginResponse, error)
 
+func newFakePlugin(inFunc fakePluginFunc, name string) fakePlugin {
```

Comment:
nit: Why not just create the fake plugin why have a new method?

Reply:
It's not really necessary. It appeared to solve a problem I was having earlier, but I think I was just adding the new field incorrectly. PR has been updated to remove this.